draft-ietf-intarea-provisioning-domains.md

```diff
@@ -640,23 +663,33 @@ listed prefixes, the associated PvD information MUST be considered
 to be a misconfiguration, and MUST NOT be used by the host. See
 {{misconfig}} for more discussion on handling such misconfigurations.
 
-## Operational Consideration to Providing the PvD Additional Information
+If the request for PvD Additional Information fails due to a TLS error,
```

Comment:
I will reply on the IETF list but this can be seen as an attacked on the PvD infra as well if an attacker is able to block TLS connections for a while. IMHO, we should probably keep trying.

Reply:
My intention in this is to indicate TLS certificate evaluation failure, not just dropping packets. So this requires either the wrong server being targeted, or else a man in the middle attack.

I may clarify by saying "fails due to a TLS certificate validation failure".

Reply:
Updated this text!